platforms/web/package.json

```diff
@@ -37,7 +37,7 @@
         "test:watch": "vitest",
         "test:ui": "vitest --ui",
         "coverage": "vitest run --coverage",
-        "update": "cd ../../bindings/wysiwyg-wasm && npm run build && cp -f ./pkg/*wysiwyg* ../../platforms/web/generated && cd ../../platforms/web && yarn"
+        "update": "cd ../../bindings/wysiwyg-wasm && npm run build && cp -f ./pkg/*wysiwyg* ../../platforms/web/generated && cd ../../platforms/web && yarn build"
```

Comment:
We should yarn instead of npm to run script since we are using yarn in the RTL

Reply:
I agree that we are in /platforms/web, but it doesn't appear to me that we are inside the /bindings/wysiwyg-wasm folder. It also says in the readme (in that folder) that `npm run build` is what to run there.

Am I missing something here? Not sure if the README should be updated or followed.

Reply:
Oh you're right, npm is used in the `wysiwyg-wasm`folder ! 

You can ignore my comment